### PR TITLE
Emit mvp post

### PR DESCRIPTION
**Pull request by rsheeter**
No tmeant to be correct, merely to tick the box for #138

### CODE REVIEW

File: fontbe/src/post.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+pub fn create_post_work() -> Box<BeWork> {
+    Box::new(PostWork {})
+}
+
+impl Work<Context, Error> for PostWork {
+    /// Generate [post](https://learn.microsoft.com/en-us/typography/opentype/spec/post)
+    fn exec(&self, context: &Context) -> Result<(), Error> {
+        // TODO a more serious post
+        let post = Post::new(
```
Comment (anthrotype) [inline fontbe/src/post.rs:22]: where is the formatType = 3.0 defined in here? Or does write-fonts take care of setting that automatically?

File: fontbe/src/post.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            Fixed::from_i32(0),
+            FWord::new(0),
+            FWord::new(16),
```
Comment (anthrotype) [inline fontbe/src/post.rs:25]: what is 16 here? I think for a viable MVP post format 3 you just need to set format to 3.0 and all the rest of the fields to 0

File: fontbe/src/post.rs (hunk 1 above)
Comment (anthrotype) [reply fontbe/src/post.rs:22]: hm I inspected the `post.table` file you compiled, and the format is 1.0, not 3.0: the first four bytes are `b'\x00\x01\x00\x00'`

Comment (anthrotype) [reply fontbe/src/post.rs:22]: post 1.0 is for fonts that contains "exactly the 258 glyphs in the standard Macintosh TrueType font file", which is not the case for you. You want formatType=3.0

File: fontbe/src/post.rs (hunk 2 above)
Comment (anthrotype) [reply fontbe/src/post.rs:25]: looks like the third new() parameter is the post.underlineThickness

https://github.com/googlefonts/fontations/blob/5fc746797c643c04b1dee2091ceb8f9642c84e37/write-fonts/generated/generated_post.rs#L79

I think 16 is too arbitrary. If you don't want to set to 0, then you can do what ufo2ft does (uses a ratio of the upem):

https://github.com/googlefonts/ufo2ft/blob/5b22f201b4650ab9ef1169357702a732e3127a5e/Lib/ufo2ft/fontInfoData.py#L313-L322

File: fontbe/src/post.rs (hunk 1 above)
Comment (rsheeter) [reply fontbe/src/post.rs:22]: Erk good catch. For some reason I thought that would get me 3 :(

File: fontbe/src/post.rs (hunk 2 above)
Comment (rsheeter) [reply fontbe/src/post.rs:25]: I switched to your earlier suggestion of defaults and filed https://github.com/googlefonts/fontmake-rs/issues/146 so I don't forget where to get better underline values from.

File: fontc/src/main.rs [hunk 3]
```diff
@@ -478,6 +484,30 @@ fn add_cmap_be_job(
     Ok(())
 }
 
+fn add_post_be_job(
+    change_detector: &mut ChangeDetector,
+    workload: &mut Workload,
+) -> Result<(), Error> {
+    if change_detector.post_be_change() {
+        let mut dependencies = HashSet::new();
+        dependencies.insert(FeWorkIdentifier::FinalizeStaticMetadata.into());
```
Comment (anthrotype) [inline fontc/src/main.rs:493]: I think your minimalist post v3 does not have any dependencies, does it? It's a constant string, basically

**Review by anthrotype — APPROVED** [review]

File: fontc/src/main.rs (hunk 3 above)
Comment (rsheeter) [reply fontc/src/main.rs:493]: That's right, will remove.

**Pull request merged**